src/views/Pools/components/PoolCard/AprRow.tsx

```diff
@@ -61,26 +67,34 @@ const AprRow: React.FC<AprRowProps> = ({ pool, stakedBalance, performanceFee = 0
     />,
   )
 
+  const isValidate = apr !== undefined && !isNaN(apr)
+
   return (
     <Flex alignItems="center" justifyContent="space-between">
       {tooltipVisible && tooltip}
       <TooltipText ref={targetRef}>{vaultKey ? `${t('APY')}:` : `${t('APR')}:`}</TooltipText>
-      {apr || isFinished ? (
-        <ApyLabelContainer alignItems="center" onClick={onPresentApyModal}>
-          <Balance
-            fontSize="16px"
-            isDisabled={isFinished}
-            value={isFinished ? 0 : apr}
-            decimals={2}
-            unit="%"
-            onClick={onPresentApyModal}
-          />
-          {!isFinished && (
-            <IconButton variant="text" scale="sm">
-              <CalculateIcon color="textSubtle" width="18px" />
-            </IconButton>
+      {isValidate || isFinished ? (
```

Comment:
We can make this component so we can use it in both places to avoid duplication

Reply:
@memoyil  Thank for your recommend. Updated.